atd-vze/src/views/Crashes/Recommendations/RecommendationMultipleSelectDropdown.js

```diff
@@ -12,8 +12,14 @@ const RecommendationMultipleSelectDropdown = ({
 }) => {
   // Trigger mutation call to add partner upon click
   const handleOptionClick = selectedItem => {
-    // Mutation expect lookup IDs as integers
-    const valuesObject = { [field]: parseInt(selectedItem.id) };
+    // If the recommendation record doesnt exist yet we need to format the object for a nested insert
+    // to create a new recommendation and a new partner in the same request
+    const valuesObject = {
+      recommendations_partners: {
+        // Mutation expects lookup IDs as integers
+        data: { [field]: parseInt(selectedItem.id) },
```

Comment:
alternatively i could have this `valuesObject` conditionally formatted based on whether `doesRecommendationRecordExist`, so it only gets this nested insert format if the rec record doesnt exist yet, and we wouldnt have to specify `valuesObject.recommendations_partners.data` in the `onAddPartner` function